app/javascript/components/pages/AboutUs.js

```diff
@@ -42,17 +42,23 @@ const AboutUs = () => {
               <Typography align="center" component="div" variant="h6">
                 Design Lead
               </Typography >
-              <Typography align="center" variant="p"
+              <Typography align="center" variant="body2"
                 component="div">
                 <br/>
                 Here's a sentence about me. Here's something I like. Here's something I wanna do.
                 <br/>
               </Typography>
               <br/>
-              <GitHubIcon><Link to="https://github.com/nathaniel-ross">Github</Link></GitHubIcon>
-                <LinkedInIcon><Link to=" https://nathaniel-ross.github.io/">Portfolio</Link></LinkedInIcon>
-                <WebAssetIcon><Link to=" https://www.linkedin.com/in/nathaniel-rossloyo/">LinkedIn</Link></WebAssetIcon>
-              </CardContent>
+              <IconButton color="primary" onClick={() => window.open('https://github.com/nathaniel-ross')}>
+                <GitHubIcon />
+              </IconButton>
+              <IconButton color="primary" onClick={() => window.open('https://www.linkedin.com/in/nathaniel-rossloyo/')}>
```

Comment:
Consider using a different method to get your links to open without directly  affecting the DOM